playbooks/roles/deploy-osh/templates/ingress-kube-system.yaml.j2

```diff
@@ -13,6 +13,6 @@ network:
     # possible options: routed, keepalived
     mode: routed
     interface: ingress-vip
-    addr: {{ suse_osh_deploy_vip_with_cidr }}
+    addr: {{ socok8s_ext_vip }}
```

Comment:
is this correct? shouldnt it be `socok8s_deploy_vip_with_cidr` ?

Reply:
This is some merge issue. I have addressed it locally but for some reason, update is not reflected here.
socok8s_ext_vip  is updated name for suse_osh_deploy_vip_with_cidr .